ultra/train.py

```diff
@@ -146,13 +152,28 @@ def train(task, num_episodes, policy_class, eval_info, timestep_sec, headless, s
     parser.add_argument(
         "--seed", help="environment seed", default=2, type=int,
     )
+    parser.add_argument(
+        "--log-dir", help="log directory location", default="logs", type=str,
+    )
     args = parser.parse_args()
 
     num_cpus = max(
         1, psutil.cpu_count(logical=False) - 1
     )  # remove `logical=False` to use all cpus
 
-    policy_class = "ultra.baselines.sac:sac-v0"
+    with open("/home/jenish/Desktop/temp/SMARTS/ultra/agent_pool.json", "r") as f:
+        objects = ijson.items(f, "agents")
+        for o in objects:
+            policy_pool = o
```

Comment:
Oh wait sorry, what does this part do here?

Reply:
The way ijson extracts the objects, I for some reason wasn't able to do just policy = objects[0], It demanded me to loop once